script/main.js

```diff
@@ -0,0 +1,59 @@
+const shelf = document.getElementById('shelf');
+const bookItem = document.getElementById('bookItem');
+
+const item = document.createElement('div');
+item.classList.add('book');
+
+const form = document.getElementById('form');
+const inputTitle = document.getElementById('title');
+const inputAuthor = document.getElementById('author');
+
+const store = localStorage.getItem('store') ? JSON.parse(localStorage.getItem('store')) : [];
+
+function removeBook(book) {
+  const bookItems = Array.from(document.getElementsByClassName('book'));
+  Object.keys(bookItems).forEach((key) => {
+    if (book === key) {
+      bookItems[key].remove();
+      store.splice(key, 1);
```

Comment:
- [OPTIONAL] I would suggest you use array methods that doesn't mutate the array, e.g. `concat`, `filter`, `slice` instead of `push`, `pop`, `splice` and so on.

Reply:
Let me, kindly, take this into account for the following projects